dist/index.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
```

Comment:
as per note above, ignore stuff in the dist file, that's transpiled build. We have no control over that.